### PR TITLE
Update documentation toolbar with Twitter and Github link

### PR DESCRIPTION
**Pull request by gsheni**
- Added html theme option to add twitter and github link

### CODE REVIEW

**Comment by gsheni** [comment]
Preview link: https://evalml.alteryx.com/en/gs_add_theme_options/

File: docs/source/conf.py [hunk 1]
```diff
@@ -96,7 +96,10 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
+html_theme_options = {
+    "github_url": "https://github.com/alteryx/evalml",
```
Comment (dsherry) [inline docs/source/conf.py:100]: Can't hurt to add this :) we do already have the github button working somehow.

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -17,6 +17,7 @@ Release Notes
         * Updated components and pipelines to return ``Woodwork`` data structures :pr:`1668`
         * Dropped support for Python version 3.6 :pr:`1751`
     * Documentation Changes
+        * Add Twitter and Github link to documentation toolbar :pr:`1754`
```
Comment (dsherry) [inline docs/source/release_notes.rst:20]: 👍

**Review by dsherry — APPROVED** [review]
Thank you @gsheni !

@tyler3991 FYI

**Pull request merged**